evmbin/src/display/json.rs

```diff
@@ -59,25 +95,37 @@ impl Informant {
 	fn informant_trace(informant: &Informant, gas_used: U256) -> String {
 		let info = ::evm::Instruction::from_u8(informant.instruction).map(|i| i.info());
 
-		json!({
-			"pc": informant.pc,
-			"op": informant.instruction,
-			"opName": info.map(|i| i.name).unwrap_or(""),
-			"gas": format!("{:#x}", gas_used.saturating_add(informant.gas_cost)),
-			"gasCost": format!("{:#x}", informant.gas_cost),
-			"memory": format!("0x{}", informant.memory.to_hex()),
-			"stack": informant.stack,
-			"storage": informant.storage,
-			"depth": informant.depth,
-		}).to_string()
+		// Reference: https://serde.rs/attr-skip-serializing.html
```

Comment:
Not sure what this comment means. Can you elaborate?

Reply:
It was just a link to a reference but it's not important so i've removed it in commit 8ab7a78189efb3c318827eb873bd5f4c8df67e51